package.json

```diff
@@ -12,9 +12,9 @@
     "watch:tsc": "tsc -w",
     "start": "rollup --config=rollup.local-dev.config.js -w -m",
     "test": "jest",
-    "lint:fix": "eslint --fix 'src/**/*.ts'",
-    "lint": "eslint 'src/**/*.ts' && echo 'no lint problems found!'",
-    "pretty": "prettier --write 'src/**/*.ts'",
+    "lint:fix": "eslint --fix src/**/*.ts && eslint --fix example/src/**/*.ts",
+    "lint": "eslint src/**/*.ts && eslint example/src/**/*.ts && echo 'no lint problems found!'",
+    "pretty": "prettier --write src/**/*.ts && prettier --write example/src/**/*.ts",
```

Comment:
I would like us to change this to "prettier --write ."  - so that it does everything including all configs - maybe you would accept that in a different PR